src/Microsoft.AspNet.Hosting/WebHostBuilder.cs

```diff
@@ -163,6 +166,17 @@ public WebHostBuilder UseEnvironment(string environment)
             return this;
         }
 
+        public WebHostBuilder UseApplication(IHttpApplication application)
```

Comment:
Not sure if this is useful but since we separated Server and HttpApplication, there maybe cases where we want to specify a non-default application?